src/Microsoft.AspNet.Server.Kestrel/compiler/preprocess/SPCM.cs

```diff
@@ -1,6 +0,0 @@
-﻿namespace Microsoft.AspNet.Server.Kestrel
-{
-    public class StandardsPoliceCompileModule : Microsoft.StandardsPolice.StandardsPoliceCompileModule
```

Comment:
Because of the target framework changes. It can't find the right dependencies in the lock file. Null ref is bad but DNX is gone anyways. 

We can turn this into an analyzer once we get support in the cli for that